ref/feature/mpJwt/examples.adoc

```diff
@@ -1,35 +1,33 @@
-The `mpJwt` feature adds the libraries that are required for MicroProfile JWT implementation.
+== Examples
+
+=== Basic configuration to implement MicroProfile JWT
 
 The `<mpJwt/>` element is required to configure the processing of the caller’s JWT.
 
+The Secure Sockets Layer(SSL)and the JSON Web Token(JWT) uses a keystore by using the maven keytool plugin.The keystore is built by using the maven keytool plugin.The applicaton uses HTTP ports and contains the user and login endpoints. The MP JWT configuration injects the caller's JWT into a ResourceScoped bean for inspection.
+
 [source,xml]
 ----
-<server description="Backend server">
-    <featureManager>
-        <feature>mpJwt-1.1</feature>
-        <feature>transportSecurity-1.0</feature>
-        <feature>jaxrs-2.1</feature>
-        <feature>jsonp-1.1</feature>
-        <feature>cdi-2.0</feature>
-        <feature>appSecurity-3.0</feature>
-    </featureManager>
-
-
-    <!-- This is the keystore that will be used by SSL and by JWT.
-         The keystore is built using the maven keytool plugin -->
     <keyStore id="defaultKeyStore" location="keystore.jceks" type="JCEKS"
               password="secret" />
-
-    <!-- The HTTP ports that the application will use. -->
     <httpEndpoint id="defaultHttpEndpoint" host="*" httpPort="${http.port}"
                   httpsPort="${https.port}"/>
-
-    <!-- The application containing the user and login endpoints. -->
     <webApplication location="${app.name}" contextRoot="/" />
-
-    <!-- The MP JWT configuration that injects the caller's JWT into a
-         ResourceScoped bean for inspection. -->
     <mpJwt id="jwtUserConsumer" keyName="default" audiences="simpleapp"
            issuer="${jwt.issuer}"/>
 </server>
 ----
+
+=== Specify login with Micrprofile JSON Web Token for only a subset of applications, URLs, browsers, or IP addresses
+
+To protect only a subset of applications, URLs, or IP addresses, use an authentication filter. The security configuration takes effect only when the conditions in the filter are met. In the following example, requests that target the `myApp` application are processed by the mp-jwt feature:
+
+[source,xml]
+----
+<mp-jwt authFilterRef="authFilter1" />
```

Comment:
does `mp-jwt` require additional config here, as in the above example? I'm not sure